.buildkite/pipeline.yml

```diff
@@ -1,4 +1,16 @@
 steps:
   - label: check trailing whitespace
     commands:
-    - .buildkite/check-trailing-whitespace.sh
+      - .buildkite/check-trailing-whitespace.sh
+  - label: backend build
```

Comment:
What is `backend build` exactly? We have `backend lib`, `backend test` and `backend build`. Is `backend build` supposed to build the library, test, everything or something else? I think there is some redundancy here.

Reply:
It builds `backend.server`, according to the way things are declared now. This file won't be used either way, as we'll use the generate pipeline steps.